OpenEdXMobile/src/main/java/org/edx/mobile/services/EdxCookieManager.java

```diff
@@ -66,7 +70,16 @@ public void clearWebWiewCookie() {
     }
 
     public synchronized  void tryToRefreshSessionCookie( ){
-        if (loginCall == null || loginCall.isCanceled()) {
+        final String userCookies = loginPrefs.getUserCookies();
+        if (userCookies != null){
+            final String[] cookies = userCookies.split(";");
```

Comment:
Should this use something like `Cookie.parseAll()` here as in line 90 in this file?

Reply:
`Cookie.parseAll()` has as parameters (HttpUrl url, Headers headers) in this case the userCookies is a String so I can not apply that method